src/openloco/localisation/string_ids.h

```diff
@@ -132,7 +135,7 @@ namespace openloco::string_ids
     constexpr string_id wcolour2_stringid2 = 457;
 
     constexpr string_id audio_device_none = 479;
-    constexpr string_id audio_device_name = 480;
+    constexpr string_id stringptr = 480;
```

Comment:
Not sure if I find this change appropriate

Reply:
It was originally used to depict audio device names, of course. The string in question simply reads `{STRING}`, though, and is the only string like it in the string table — all others have more code points.

I think generalising it is appropriate.